src/components/not-found/NotFound.scss

```diff
@@ -0,0 +1,33 @@
+
+.errorContainer{
+  height:92vh;
+  :before{
+    content: "";
+    position: absolute;
+    top: 15%;
+    left: 22.5%;
+    width: 60%;
+    height: 80%;
```

Comment:
sprawdzales na roznych ekranach jak sie to prezentuje? Troche sie boje ze te style na sztywno moga roznie sie zachowac